test/chi/custom-elements/text-input.pug

```diff
@@ -0,0 +1,251 @@
+---
+title: Text inputs
+---
+script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
+script(nomodule='' src='../../js/ce/ux-chi-ce/ux-chi-ce.js')
+
+each size in ['default', 'sm', 'md', 'lg', 'xl']
+  div(class=`test-${size}`)
+    span.chi-h2= `Text Input / ${size}`
+    table(cellpadding="0", cellspacing="0", width="1200px")
+      each status in ['initial', 'hover', 'focus', 'disabled']
+        tr
+          td
+            small= `${status}`
+          each state in ['default', 'success', 'warning', 'danger']
+            td
+              .-m--1
+                chi-text-input(
+                  state=state=='default'?'':state,
+                  status=status,
+                  size=size=='default'?'':size,
+                  value='Value',
+                  disabled=status=='disabled'?'disabled':false)
```

Comment:
Can status take the 'disabled' value? Can't we just use
```
                  disabled=disabled
```
Please, check the rest of the file for further occurrences

Reply:
As I already said, I would remove the status Prop. I will create a new disabled prop and one status internal State or one internal State for each of `active`, `focus` and `hover` statuses.